uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/IdentityZoneEndpointsIntegrationTests.java

```diff
@@ -246,6 +250,60 @@ public void testUpdateZoneWithDisableClientRedirectUri() throws Exception {
         assertTrue(updateResponse.getBody().isEnableRedirectUriCheck());
     }
 
+    @Test
+    public void testDeletePortedZone() {
+        //Create Identity Zone
+        IdentityZone idZone = new IdentityZone();
+        String id = UUID.randomUUID().toString();
+        idZone.setId(id);
+        idZone.setSubdomain(id);
+        idZone.setName("testCreateZone() " + id);
+        ResponseEntity<Void> response = client.exchange(serverRunning.getUrl("/identity-zones"), HttpMethod.POST, new HttpEntity<>(idZone), new ParameterizedTypeReference<Void>() {
+        }, id);
+        assertEquals(HttpStatus.CREATED, response.getStatusCode());
+
+        //Port Zone and Validate
+        ResponseEntity<Void> importResponse = portZone(id);
+        OrchestratorZoneResponse expectedResponse = new OrchestratorZoneResponse();
+        expectedResponse.setName(zoneId);
+        expectedResponse.setMessage(ZONE_CREATED_MESSAGE);
+        expectedResponse.setState(OrchestratorState.CREATE_IN_PROGRESS.toString());
+
+        Assertions.assertEquals(HttpStatus.ACCEPTED, importResponse.getStatusCode());
+
+        ResponseEntity<Void> deleteResponse = client.exchange(serverRunning.getUrl("/identity-zones/" + id), HttpMethod.DELETE, null, new ParameterizedTypeReference<Void>() {
+        });
+
+        Assertions.assertEquals(deleteResponse.getStatusCode(), HttpStatus.UNPROCESSABLE_ENTITY);
+    }
+
+    private ResponseEntity<Void> portZone(String zoneId) {
+        ClientCredentialsResourceDetails orchestratorZoneClientCredentials = new ClientCredentialsResourceDetails();
+        orchestratorZoneClientCredentials.setClientId("orchestrator-zone-provisioner");
+        orchestratorZoneClientCredentials.setClientSecret("orchestratorsecret");
+        orchestratorZoneClientCredentials.setId("orchestrator-zone-provisioner");
+        orchestratorZoneClientCredentials.setAccessTokenUri("http://localhost:8080/uaa/oauth/token");
+        OAuth2RestTemplate restTemplate = new OAuth2RestTemplate(orchestratorZoneClientCredentials, new DefaultOAuth2ClientContext());
+        String requestBody = JsonUtils.writeValueAsString(getOrchestratorImportZoneRequest("TestZone", "admin-secret", "TestCone", zoneId));
```

Comment:
minor typo error, change TestCone to TestZone

Reply:
corrected